starboard/android/apk/app/src/main/java/dev/cobalt/coat/StarboardBridge.java

```diff
@@ -799,4 +800,24 @@ protected boolean getIsAmatiDevice() {
   protected String getBuildFingerprint() {
     return Build.FINGERPRINT;
   }
+
+  @SuppressWarnings("unused")
+  @UsedByNative
+  protected long getPlayServicesVersion() {
+    try {
+      if (android.os.Build.VERSION.SDK_INT < 28) {
+        return appContext
+            .getPackageManager()
+            .getPackageInfo(GOOGLE_PLAY_SERVICES_PACKAGE, 0)
+            .versionCode;
+      } else {
+        return appContext
+            .getPackageManager()
+            .getPackageInfo(GOOGLE_PLAY_SERVICES_PACKAGE, 0)
+            .getLongVersionCode();
+      }
+    } catch (Exception e) {
+      return 0;
```

Comment:
Could you `Log.e` or `Log.w` here please, when this doesn't work ? Quietly swallowing a very broad exception will cause some hard debugging at some point. 

Ideally we could narrow the exception type down a little as well, but it's okay as first pass.